README.md

```diff
@@ -145,4 +145,41 @@ The above script builds the necessary docker images and starts the SpringBoot se
 The [ITK Testbench](https://digital.nhs.uk/services/interoperability-toolkit/developer-resources/itk-test-centre/itk-testbench) provided by NHS Digital has an essential tool for ITK interface development and conformance.  This tool provides resources that will allow you to adequately test the 111 adaptor within your own environment.
 
 ## Test Pack
-Our testing pack as well our test harness can be found inside the test-suite directory [Test-Suite](./test-suite)
\ No newline at end of file
+Our testing pack as well our test harness can be found inside the test-suite directory [Test-Suite](./test-suite)
+
+## Running Test Harness
+Pre-Requisites:
+* Install Java 17
+* Install Docker
+* Install a preferred IDE
+
+1. Run docker desktop
+2. Navigate to project directory via IDE Terminal
+3. cd into docker folder
+4. run command ./start-local-environment.sh (On Mac or Linux use ./start-local-environment-arm.sh)
+5. Wait for command to finish building
+6. Docker desktop should have something like this(add in docker hierarchy)
+7. cd into test-suite folder
+8. run command ./start-local.sh (this will build the test harness)
+9. Test harness will open in default browser
+
+### Using the test harness:
+- The test harness has data pre-populated in each form - clicking reset will undo any alternations
```

Comment:
We should include something on how they point the test harness to run against a custom hosted instance.